cl/opinion_page/templates/view_opinion.html

```diff
@@ -104,16 +104,17 @@ <h4>
         {% endif %}
 
         {# Show cases this case cites #}
-        {% if top_authorities %}
+        {% if authorities_count > 0  %}
             <div id="authorities" class="sidebar-section">
-                <h3><span>Authorities ({{ cluster.authority_count }})</span></h3>
+                <h3><span>Authorities ({{ authorities_count }})</span></h3>
                 <p class="bottom">This opinion cites:</p>
                 <ul>
                     {% for authority in top_authorities %}
                         <li>
                             <a href="{{ authority.get_absolute_url }}?{{ request.META.QUERY_STRING }}">
                                 {{ authority.caption|safe|truncatewords:10|v_wrapper }}
                             </a>
+                            <span class="small gray">({{ authority.citation_depth }} time{{ authority.citation_depth|pluralize }})</span>
```

Comment:
Here and in one other place I'll mark, we need to have an `if` that makes sure `citation_depth` isn't `None`. This could happen if the search engine has an item that has been removed from the DB. That shouldn't happen, but it can and it sometimes does for brief periods of time. When it does, `get_citation_depth_between_clusters` will return `None` (I checked).

Reply:
Fixed in 4d842b2